run/jobs/e2e_test.py

```diff
@@ -80,15 +81,20 @@ def test_end_to_end(setup_job):
         "-protoPayload.serviceName=\"run.googleapis.com\""
     )
 
-    # Retry a maximum number of 10 times to find results in stackdriver
+    # Retry a maximum number of 5 times to find results in Cloud Logging
     found = False
-    iterator = client.list_log_entries(
-        {"resource_names": resource_names, "filter": filters}
-    )
-    for entry in iterator:
-        if "Task" in entry.text_payload:
-            found = True
-            # If there are any results, exit loop
+    for x in range(5):
+        iterator = client.list_log_entries(
+            {"resource_names": resource_names, "filter": filters}
+        )
+        for entry in iterator:
```

Comment:
Nit: should we make this shorter?

Untested example code:
```
found = any([x for x in iterator if "Task" in entry.text_payload])
```

Reply:
Depends on the number of entries in the iterator, but I could see both being good to have (shorter version or the current version)